pkg/tsdb/prometheus/prometheus.go

```diff
@@ -216,3 +217,16 @@ func parseResponse(value model.Value, query *PrometheusQuery) (*tsdb.QueryResult
 
 	return queryRes, nil
 }
+
+func IsAPIError(err error) bool {
+	var e *apiv1.Error
+	return errors.As(err, &e)
+}
+
+func WrapAPIError(err error) error {
```

Comment:
Better naming since the error isn't wrapped
```suggestion
func ConvertAPIError(err error) error {
```